packages/example-app-base/src/components/App/App.tsx

```diff
@@ -121,4 +116,15 @@ export class App extends React.Component<IAppProps, any> {
   private _onLinkClick(): void {
     this.setState({ isMenuVisible: false });
   }
+
+  private _onRenderLink(link: INavLink): (JSX.Element | null)[] {
```

Comment:
This function should either have `@autobind` or be a free-function, not bound to the class.